front/src/components/FilmListItem/cinemaSeanceInfo.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import {NavLink} from 'react-router-dom'
+
+const CinemaSeanceInfo = ({
+    filmInfo,
+}) => {
+    console.log(filmInfo);
```

Comment:
There shouldn't be any console.log in master branch (only in some rare cases, I doubt it is such case)